joint1_calibration: -131.4697265625
joint2_calibration: -183.3984375
joint3_calibration: -243.1640625
joint4_calibration: 4
